instruqt-tracks/secure-service-networking-for-aws/03-create-vpcs/assignment.md

```diff
@@ -11,7 +11,7 @@ notes:
 tabs:
 - title: Infrastructure Overview
   type: website
-  url: https://htmlpreview.github.io/?https://raw.githubusercontent.com/hashicorp/field-workshops-consul/master/instruqt-tracks/secure-service-networking-for-aws/assets/images/ssn4aws-infra-overview.html
+  url: https://htmlpreview.github.io/?https://raw.githubusercontent.com/hashicorp/field-workshops-consul/ssnfa-stability/instruqt-tracks/secure-service-networking-for-aws/assets/images/ssn4aws-infra-vpc.html
```

Comment:
This should probably remain a reference to the default branch -- these files are getting merged into the master branch, correct?

Reply:
Yes.  Is the best way to reference the default branch to use "master" or is there another way?

https://htmlpreview.github.io/?https://raw.githubusercontent.com/hashicorp/field-workshops-consul/**master**/instruqt-tracks/secure-service-networking-for-aws/assets/images/ssn4aws-infra-vpc.html